spring-cloud-gcp-logging/README.adoc

```diff
@@ -72,3 +76,33 @@ Your configuration may then look something like this:
 `STACKDRIVER` appender.
 
 Also see the link:../spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging[spring-cloud-gcp-starter-logging] module.
+
+=== Log via Console
```

Comment:
Why in the README? 
Why isn't this reflected in the Reference Manual?

Thanks

Reply:
How to update the Reference Manual in this PR, it is not part of this project isn't it?

Reply:
There is a `docs` module in this project. That's where all the `.adoc` s are hanging out. 

That's still unclear for me why support READMEs in each module where it is easy to lose the track and have inconsistency...